.github/workflows/publish-site.yaml

```
@@ -0,0 +1,16 @@
+name: publish-site
+
+on:
+  # Trigger the workflow on push to main branch
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: ProjectPythia/cookbook-actions/.github/workflows/build-book.yaml@main
```

This should be the correct way to call the `build_book.yaml` workflow in this case:
```suggestion
    uses: ProjectPythia/cookbook-actions/.github/workflows/build-book.yaml@main
    with: 
        path_to_notebooks: 'portal'
        base_url: ''
```